### PR TITLE
create java spring project template

### PR DESCRIPTION
**Pull request by Mr-Keltt**

### CODE REVIEW

File: user-service/HELP.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Getting Started
```
Comment (savin000) [inline user-service/HELP.md:1]: Remove this please. We don't need this autogenerated help.md file

File: user-service/mvnw [hunk 2]
```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
```
Comment (savin000) [inline user-service/mvnw:1]: Why do we need this script? Let's check on meeting

File: user-service/mvnw.cmd [hunk 3]
```diff
@@ -0,0 +1,188 @@
+@REM ----------------------------------------------------------------------------
```
Comment (savin000) [inline user-service/mvnw.cmd:1]: Same here as above. No need in this generated script

File: user-service/user-service.iml [hunk 4]
```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (savin000) [inline user-service/user-service.iml:1]: Please add .gitignore file to project and add THIS file there to delete if from code base

**Review by savin000 — COMMENTED** [review]
Add a test controller please to check if everything works fine

**Comment by Mr-Keltt** [comment]
It seems like I fixed everything.

File: user-service/pom.xml [hunk 5]
```diff
@@ -1,41 +1,41 @@
...
-    </properties>
-    <dependencies>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter</artifactId>
-        </dependency>
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
```
Comment (savin000) [inline user-service/pom.xml:6]: remove this long tabs please

File: user-service/pom.xml [hunk 6]
```diff
@@ -1,41 +1,41 @@
...
+		<artifactId>spring-boot-starter-parent</artifactId>
+		<version>3.0.2</version>
+		<relativePath/> <!-- lookup parent from repository -->
+	</parent>
+	<groupId>ru.vsu.csd</groupId>
+	<artifactId>user-service</artifactId>
+	<version>0.0.1-SNAPSHOT</version>
+	<name>user-service</name>
+	<description>Recommendation system</description>
```
Comment (savin000) [inline user-service/pom.xml:15]: Write a short description here. Something like "A service to auth user and initiate a voting"

File: user-service/src/main/java/ru/vsu/csd/userservice/UserServiceApplication.java [hunk 7]
```diff
@@ -6,8 +6,8 @@
 @SpringBootApplication
```
Comment (savin000) [inline user-service/src/main/java/ru/vsu/csd/userservice/UserServiceApplication.java:6]: Seems like u opened this in another text editor and it edited everything with 1 tab == 8 spaces. Make 1 tab = 4 spaces pls in all files

**Review by savin000 — COMMENTED** [review]
Add 1 test controller please to ensure that everything works. I mean something like GET that responses a "hello world" string

File: user-service/pom.xml (hunk 5 above)
Comment (Mr-Keltt) [reply user-service/pom.xml:6]: I don't know what you mean

**Comment by Mr-Keltt** [comment]
I suppose github draws tabs for each user as configured by a particular user. When cloning a repository to a computer, all tabs are in order.
But it is also possible, this can be changed in the project settings. I couldn't find any information about this.

**Comment by savin000** [comment]
> I suppose github draws tabs for each user as configured by a particular user. When cloning a repository to a computer, all tabs are in order. ([Read this article](https://docs.github.com/ru/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/managing-your-tab-size-rendering-preference))
> But it is also possible, this can be changed in the project settings. I couldn't find any information about this.

You were right

File: .idea/.gitignore [hunk 8]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (savin000) [inline .idea/.gitignore:1]: add all files inside .idea to gitignore and remove them from here

File: user-service/.mvn/wrapper/maven-wrapper.properties [hunk 9]
```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.8.6/apache-maven-3.8.6-bin.zip
```
Comment (savin000) [inline user-service/.mvn/wrapper/maven-wrapper.properties:1]: Why do we need .mvn/ folder and items inside it? If no need, delete them please, or explain

File: user-service/pom.xml [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+        </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
```
Comment (savin000) [inline user-service/pom.xml:46]: Why spring-boot-starter-web dependency is duplicated? Remove one please.

File: user-service/user-service.iml [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
```
Comment (savin000) [inline user-service/user-service.iml:1]: add all .iml files to gitignore. Remove it from git please

**Review by savin000 — COMMENTED** [review]
Please add target/ folder to gitignore and remove it and all its items from git

**Review by savin000 — APPROVED** [review]
Everything is ok

**Pull request merged**